src/Illuminate/Auth/Access/Gate.php

```diff
@@ -117,7 +117,7 @@ public function define($ability, $callback)
     protected function buildAbilityCallback($callback)
     {
         return function () use ($callback) {
-            list($class, $method) = explode('@', $callback);
+            list($class, $method) = Str::parseCallback($callback, null);
```

Comment:
might be an idea to make the default 2nd param `null`, then avoid having to use `null` everywhere we call it?